scripts/rtinst

```diff
@@ -243,7 +243,13 @@ if [ "$relno" = "20" ] || [ "$(lsb_release -sr)" = "20.04" ] || [ "$relno" = "11
   libcver=libcurl4
 fi
 
-if [ "$relno" = "21" ] || [ "$(lsb_release -sr)" = "21.04" ]; then
+if [ "$relno" = "21" ] || [ "$(lsb_release -sr)" = "21.10" ]; then
+  phpver=php7.4
```

Comment:
Ubuntu 22.04 doesn't have php7.4 on their repository. So, it's good to use php8.1 instead of php7.4